cachito/workers/celery_logging.py

```diff
@@ -0,0 +1,111 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import inspect
+import logging
+
+from celery.utils.log import ColorFormatter
+
+
+class RequestFilter(logging.Filter):
+    """A log filter that sets ``request_id`` on the log record."""
```

Comment:
Why are we using  `` (two of them) here ?

Reply:
This is because the docstrings are in the reStructuredText format:
https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup